### PR TITLE
Add possibility to load fonts from the file system

### PR DESCRIPTION
**Pull request by AlexandrRzz**
https://trello.com/c/BVY0NOjm/19729-introduce-supporting-a-fixed-layout-in-reader

### CODE REVIEW

File: platforms/js/FontLoader.hx [hunk 1]
```diff
@@ -32,6 +32,13 @@ class FontLoader {
 		untyped __js__("setTimeout(onDone, 25)");
 	}
 
+	public static function loadFSFont(family : String, url : String) {
+		addFontFace("g_font_12", "./php/filesystemdb.php?operation=download_blob_binary&replica=true&file_id=122&cached=0&filename=g_font_12.ttf&thumbnail=0&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6IjEiLCJlbWFpbCI6ImFkbWluQGFyZWE5LmRrIiwiZmlyc3ROYW1lIjoiQWRtaW4iLCJsYXN0TmFtZSI6IkFjY291bnQiLCJpc3MiOiJjdXJhdG9yIiwiaWF0IjoxNjY5ODk0NzY0LCJleHAiOjE2NzExMDQzNjR9.n6UJ08ePX6CVZvFSsxolS30HiEfydEWlX1531kxr6kU&accessKey=RmlsZXN5c3RlbUluZm8uMC5jcnVkLjEyMi4yMDIyMTIzMS4x.5c833e2bb6eee6773dd648a1a23b273c");
```
Comment (ykuzmenko73) [inline platforms/js/FontLoader.hx:36]: why the URL is hardcoded? pass all you need as arguments

File: platforms/js/FontLoader.hx [hunk 2]
```diff
@@ -32,6 +32,13 @@ class FontLoader {
...
+		addStyledText("g_font_12");
+		addFontFace("g_font_11", "./php/filesystemdb.php?operation=download_blob_binary&replica=true&file_id=123&cached=0&filename=g_font_11.ttf&thumbnail=0&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6IjEiLCJlbWFpbCI6ImFkbWluQGFyZWE5LmRrIiwiZmlyc3ROYW1lIjoiQWRtaW4iLCJsYXN0TmFtZSI6IkFjY291bnQiLCJpc3MiOiJjdXJhdG9yIiwiaWF0IjoxNjY5ODk0NzY0LCJleHAiOjE2NzExMDQzNjR9.n6UJ08ePX6CVZvFSsxolS30HiEfydEWlX1531kxr6kU&accessKey=RmlsZXN5c3RlbUluZm8uMC5jcnVkLjEyMy4yMDIyMTIzMS4x.af73bc7e329a44c371f311405c2b3db6");
```
Comment (ykuzmenko73) [inline platforms/js/FontLoader.hx:38]: looks not correct to load two fonts in one call for just one

**Review by ykuzmenko73 — COMMENTED** [review]
small comments

**Comment by alstrup** [comment]
It is wrong to have any Rhapsode-specific stuff in flow9 repo, so make sure that all the URL can be given from the outside and reside in the Rhapsode or innovation repos.

**Review by ykuzmenko73 — APPROVED** [review]

**Pull request merged**